auditbeat/module/file_integrity/event_linux.go

```diff
@@ -0,0 +1,179 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//go:build linux
+
+package file_integrity
+
+import (
+	"os"
+	"os/user"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/elastic/ebpfevents"
+)
+
+// NewEventFromEbpfEvent creates a new Event from an ebpfevents.Event.
+func NewEventFromEbpfEvent(
+	ee ebpfevents.Event,
+	maxFileSize uint64,
+	hashTypes []HashType,
+	fileParsers []FileParser,
+	isExcludedPath func(string) bool,
+) (Event, bool) {
+	var (
+		path, target string
+		action       Action
+		metadata     Metadata
+		err          error
+	)
+	switch ee.Type {
+	case ebpfevents.EventTypeFileCreate:
+		action = Created
+
+		fileCreateEvent := ee.Body.(*ebpfevents.FileCreate)
+		path = fileCreateEvent.Path
+		if isExcludedPath(path) {
+			event := Event{Path: path}
+			return event, false
+		}
+		target = fileCreateEvent.SymlinkTargetPath
```

Comment:
This overall looks good, just one minor question for my understanding, if the file created is not a symlink will ```.SymlinkTargetPath``` point to the same path as ```.Path```

Reply:
no, it will be empty